09-features.md

```diff
@@ -64,10 +64,14 @@ The origin node:
     unless indicated that it must set the odd feature bit instead.
   * MUST NOT set feature bits it does not support.
   * MUST NOT set feature bits in fields not specified by the table above.
+  * MUST NOT set both the optional and mandatory field.
   * MUST set all transitive feature dependencies.
+  * MUST support:
+    * `var_onion_optin`
 
-The origin node MUST support:
-  * `var_onion_optin`
+The received node:
+  * if both the optional and the mandatory feature bits are set,
```

Comment:
nittynit: if both the optional and the mandatory feature its **in a pair** are set ...

Reply:
Thanks!